src/GeoIp2/WebService/Client.php

```diff
@@ -180,7 +188,14 @@ private function responseFor($endpoint, $class, $ipAddress)
 
         $client = $this->guzzleClient ?
             $this->guzzleClient : new GuzzleClient();
-        $request = $client->get($uri, array('Accept' => 'application/json'));
+        $options = array();
+        if($this->timeout != null){
```

Comment:
Done, good catch. Though specifying a timeout of 0 or false seems logically equivalent to me :confused: